react/week4/react-w4/src/Results.js

```diff
@@ -0,0 +1,18 @@
+import { useContext } from "react";
+import { UsersContext } from "./App";
+
+export default function Results() {
+  const { loading, queryResult } = useContext(UsersContext);
+  return (
+    <>
+      {loading === true && <p>Loading...</p>}
```

Comment:
Hi @Sarulathaanbu please try to meet this requirement as well. "When the search is failed, you should show "error fetching (reason why it failed)..."